scripts/pkglib.sh

```diff
@@ -164,6 +164,33 @@ function checkout_repo()
     cp ../scripts/apport_hook.py $OBS_WORKDIR/$APPORT_HOOK
 }
 
+function generate_kernel_config() {
+    local kernel_version=$1
+    local kernel_filename=$2
+    local kernel_url=https://www.kernel.org/pub/linux/kernel/v4.x/linux-"${kernel_version}".tar.xz
+
+    local tmpdir=/var/tmp
+    local kernel_tree_dir="${tmpdir}/linux-${kernel_version}"
+
+    # Download the kernel source tree tarball and decompress it in ${tmpdir}
+    [ ! -e "${kernel_tree_dir}.tar.xz" ] &&
+        curl -o "${kernel_tree_dir}.tar.xz" -L $kernel_url || true
+    tar -C $tmpdir -xf "${kernel_tree_dir}.tar.xz"
+
+    # Generate a new kernel config file based on the one in this repo
+    cp kernel-config-* "${kernel_tree_dir}/.config"
```

Comment:
This would be safer as `cp kernel-config-${KR_LTS}*` to guard against the possibility of copying multiple `kernel-config-${version}` files into `.config`.